R/update-version.R

```diff
@@ -55,3 +54,109 @@ get_date <- function() {
   author_time <- parsedate::parse_iso_8601(Sys.getenv("GIT_COMMITTER_DATE"))
   as.Date(author_time)
 }
+
+fledge_bump_version <- function(desc, which) {
+  version <- desc$get_version()
+  version_components <- get_version_components(version)
+
+  if (which %in% c("patch", "minor", "major", "dev")) {
+    version_components[["dev"]] <- switch(
```

Comment:
I found it easier to follow to use switch once per component (and to still separate the "usual" cases from pre-minor and pre-major).